backend/canisters/group/api/can.did

```diff
@@ -372,7 +317,6 @@ type OptionalGroupPermissions =
         delete_messages: opt PermissionRole;
         update_group: opt PermissionRole;
         pin_messages: opt PermissionRole;
-        invite_users: opt PermissionRole;
```

Comment:
Surely the `invite_users` permission is still needed?

Reply:
Yes it got accidentally removed from the candid